model/map/src/main/java/org/keycloak/models/map/client/MapClientProvider.java

```diff
@@ -245,6 +245,13 @@ public ClientModel getClientById(String id, RealmModel realm) {
             return null;
         }
 
+        //TODO: replace with Pattern/Matcher or find a better place to check if clientUuid is in uuid format
```

Comment:
the problem is that from rest services can come request with id in non-uuid format and it throws IllegalArgumentException.

Reply:
This introduces silent ignoring of an error. Throwing an error (L 257) should be thus kept.